lib/models/viewmodels/converters/tweet_v2_to_tweet_vm_converter.dart

```diff
@@ -220,10 +222,12 @@ class TweetV2ToTweetVMConverter {
   }
 
   int get _favoriteCount => tweet.publicMetrics.likeCount;
+  int get _repliesCount => tweet.publicMetrics.replyCount;
```

Comment:
I think for consistency you can add line break between _favoriteCount and _repliesCount